bin/test_snap.cmd

```diff
@@ -0,0 +1,61 @@
+@ECHO OFF 
+setlocal enabledelayedexpansion
+
+set branch=%1
+
+REM Check if parameters are set
+if %branch%.==. (
+	GOTO args_error
+) else (
+	GOTO set_env
+)
+
+
+:args_error
+echo User error: must specify branch
+echo test_snap "<branch>"
+echo Branch can be any of: 7.1, 7.2, 7.3, 7.4
+exit /b
+
+:set_env
+REM set important env vars
+IF DEFINED PFTT_SHELL GOTO :skip_set_env
+CALL %~dp0set_env.cmd
+:skip_set_env
+
+if not exist %PHP_BUILDS% (
+	md %PHP_BUILDS%
+)
+
+SET cpu[0]=x64
+SET cpu[1]=x86
+SET cpu[2]=x64
+SET cpu[3]=x86
+SET thread[0]=NTS
+SET thread[1]=NTS
+SET thread[2]=TS
+SET thread[3]=TS
+set test_pack=php-test-pack-%branch%-latest
+
+if %branch%==7.1 (
```

Comment:
I noticed `auto_test_snap.cmd` does not include 7.1 anymore. Assuming we do not test snapshots of 7.1 anymore, we might as well remove this and just `set build = vc15` without having to check the build version.

Reply:
Fixed.